src/test/java/teammates/test/pageobjects/AppPage.java

```diff
@@ -805,7 +805,7 @@ public static String processPageSourceForFailureCase(String content) {
     
     private static String processPageSourceForGodMode(String content) {
         Date now = new Date();
-        assertEquals(new SimpleDateFormat("EEE, dd MMM yyyy, HH:mm").format(now), TimeHelper.formatTime(now));
+        assertEquals(new SimpleDateFormat("EEE, dd MMM yyyy, hh:mm a").format(now), TimeHelper.formatTime12H(now));
```

Comment:
I think it ensures that if there is a change in the time format, the dev will remember to change godmode too?